worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -511,7 +511,8 @@ class FetchHandler(
       client: TransportClient,
       streamChunkSlice: StreamChunkSlice,
       req: RequestMessage): Unit = {
-    logDebug(s"Received req from ${NettyUtils.getRemoteAddress(client.getChannel)}" +
+    val remoteAddr = NettyUtils.getRemoteAddress(client.getChannel)
```

Comment:
Make it a lazy val

Reply:
Nice catch, this object is only used in some scenarios.